ironfish/src/consensus/verifier.ts

```diff
@@ -66,10 +66,14 @@ export class Verifier {
       const fee = transactionFees[i]
 
       // Miner's fee should be only the first transaction
-      if ((i === 0 && fee > 0) || (i !== 0 && fee < 0)) {
+      if (i === 0 && fee > 0) {
         return { valid: false, reason: VerificationResultReason.INVALID_MINERS_FEE }
```

Comment:
This could potentially be something like MINERS_FEE_EXPECTED if changing the verification reason makes it easier to test. I don't really have a strong opinion here

Reply:
Done